tajo-catalog/tajo-catalog-server/src/main/java/org/apache/tajo/catalog/store/AbstractDBStore.java

```diff
@@ -1110,12 +1113,20 @@ private void setProperties(final int tableId, final KeyValueSetProto properties)
     }
   }
 
-  private void unsetProperties(final int tableId, final PrimitiveProtos.StringListProto propertyKeys) {
+  private void unsetProperties(final int tableId, final PrimitiveProtos.StringListProto propertyKeys)
+      throws UnremovableTablePropertyException {
     final String deleteSql = "DELETE FROM " + TB_OPTIONS + " WHERE TID=? AND KEY_=?";
 
     Connection conn;
     PreparedStatement pstmt = null;
 
+	  Set<String> keys = Sets.newHashSet(propertyKeys.getValuesList());
```

Comment:
Please fix indentation in this code block
https://cwiki.apache.org/confluence/display/TAJO/Code+Style
Thanks